packages/cli-lib/src/gts_extractor.ts

```diff
@@ -0,0 +1,19 @@
+// @ts-ignore
+import {transform, parseTemplates} from 'ember-template-tag'
+import {parseFile as parseHbsFile} from './hbs_extractor'
+import {parseScript} from './parse_script'
+
+export function parseFile(source: string, fileName: string, options: any) {
+  const scriptParseFn = parseScript(options, fileName)
+  const {output: transformedSource} = transform({
+    input: source,
+    relativePath: '',
+  })
+
+  scriptParseFn(transformedSource)
+
+  // extract template from transformed source to then run through hbs processor
+  const [templateSource] = parseTemplates(source, '')
```

Comment:
gts can have multiple templates, so does this concatenate all of the different templates?

For example, looking at this type from `content-tag`: https://github.com/embroider-build/content-tag/?tab=readme-ov-file

We have:
```js
import { Preprocessor } from 'content-tag';
let p = new Preprocessor();

// ...
let output = p.parse(source);

for (let parseResult of output) {
  parseHbsFile(parseResult.contents, filename, options);
}
```